### PR TITLE
Jashpatel1/themecolors

### LINKED ISSUE #672: [NG] Change from pink to layer5 brand colours
**Desired Situation**
The layer5ng site is having pink colour in most of its pages. It should be changed to layer5 brand colours.

I think changes in most places would take place if the theme colours in [this folder](https://github.com/layer5io/layer5/tree/layer5-ng/src/theme) are changed. 

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by Jashpatel1**
**Description**

This PR fixes #672 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Navigation/navigation.style.js [hunk 1]
```diff
@@ -107,10 +104,7 @@ const NavigationWrap = styled.header`
             display: block;
             padding: 5px 15px;
             &:hover {
-              color: ${(props) =>
-        props.theme.menuHoverColor
-            ? props.theme.menuHoverColor
-            : "#FB7B81"} !important;
+              color: #00b39f; !important;
```
Comment (Tanuj22) [inline src/sections/Navigation/navigation.style.js:107]: Don't hardcode the colors.

File: src/sections/Navigation/navigation.style.js [hunk 2]
```diff
@@ -136,17 +130,11 @@ const NavigationWrap = styled.header`
           width: 20px;
           height: 1px;
           opacity: 0;
-          background: ${(props) =>
-        props.theme.menuHoverColor
-            ? props.theme.menuHoverColor
-            : "#FB7B81"};
+          background: #00b39f;
```
Comment (Tanuj22) [inline src/sections/Navigation/navigation.style.js:133]: same

File: src/sections/Navigation/navigation.style.js [hunk 3]
```diff
@@ -136,17 +130,11 @@ const NavigationWrap = styled.header`
...
           transition: 450ms all;
         }
         &:hover {
-          color: ${(props) =>
-        props.theme.menuHoverColor
-            ? props.theme.menuHoverColor
-            : "#FB7B81"};
+          color: #00b39f;
```
Comment (Tanuj22) [inline src/sections/Navigation/navigation.style.js:137]: same

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
Also change the heading colors from blue. Try the color for smp blue

**Comment by Tanuj22** [comment]
@leecalcote @Nikhil-Ladha @chandrashritii thoughts here. I don't think the SMP Blue color for the headings is not quite fitting here

**Comment by Nikhil-Ladha** [comment]
I think this color will fit good rgb(59 73 79)

**Comment by leecalcote** [comment]
> I think this color will fit good rgb(59 73 79)

The color of RGB (60, 73, 78), which is Hex #3C494E (that comes from layer5.io/brand) is close to the one that you're suggesting, @Nikhil-Ladha and might be a good one to try.

**Comment by leecalcote** [comment]
Soon, we will be able to refer to the [Layer5 Style Guide](https://docs.google.com/presentation/d/12XWzUN3MqCm4jchV4RWgkqikPbgkKeTDhipVufNU2BU/edit#slide=id.g984d490928_2_193).

**Comment by Nikhil-Ladha** [comment]
Oh, I meant that one only, just picked it from the chrome dev 😅

**Comment by leecalcote** [comment]
I updated the `headingColor` of each of the themes. It's an improvement, but may not be what we keep in the end. Let's give some color some time to settle in and see if it has a lasting fit.

**Review by leecalcote — APPROVED** [review]

**Pull request merged**